cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -1025,6 +1028,43 @@ To generate a QR code for multiple end devices:
 			return nil
 		}),
 	}
+	endDevicesExternalJSCommand = &cobra.Command{
+		Use:     "external-join-server [application-id] [device-id]",
```

Comment:
We want our CLI commands to be actions, and `external-join-server` is not an action. Maybe `use-external-join-server`? Aliases can be `use-external-js`, `use-ext-js`.